README.md

```diff
@@ -1,6 +1,201 @@
 Job Restrictions Plugin for [Jenkins CI][1]
 =======================
-See [Job Restrictions Plugin Page][2] for more information.
+
+| Plugin Information |
+|---------------------------------------------------------------------------------------------------------------|
+| View Job Restrictions [on the plugin site](https://plugins.jenkins.io/job-restrictions) for more information. |
```

Comment:
```suggestion
```

This is natively handled in https://plugins.jenkins.io/job-restrictions so unncessary now.

Reply:
Thanks for the feedback! Removed this section.